common/config/data.tsx

```diff
@@ -13,11 +13,11 @@ export const N_FACTOR = 8192;
 export const ANNOUNCEMENT_TYPE = 'warning';
 export const ANNOUNCEMENT_MESSAGE = (
   <div>
-    This is an Beta build of MyCrypto v4. Please only use for testing, or use production at{' '}
+    This is an Beta build of MyCrypto Beta. Please only use for testing, or use production at{' '}
```

Comment:
*a Beta build. And maybe this is just me, but a beta build of the beta reads a little redundantly to me.

Reply:
Maybe "pre-release" build or "early" build or "preview" build.

Reply:
definitely, good catch